### DISCUSSION TITLE
阿里云盘 Open 上传文件后出现很多 ffs_Lock 后缀文件

### DISCUSSION THREAD

**Discussion by hiforrest** [AlistGo/alist#3705] [General]
![Snipaste_2023-03-04_16-06-16](https://user-images.githubusercontent.com/2953848/222884837-a7593490-fb91-4ca2-bb5e-4f6bfd59e58f.png)

**Comment by Zliy997** [comment]
<img width="723" alt="image" src="https://user-images.githubusercontent.com/67785977/222891361-2bb0cbaf-6ebf-4c33-881a-f543e2226361.png">
我的情况是新增很多._Name的文件😅😅

**Comment by anwen-anyi** [comment]
没发现有这个问题啊？楼上的两位确定是因为 open上传导致的？怎么复现一下？我也用open上传过 没发现这样的情况吖

**Reply by hiforrest** [reply]
是用alist+Raidriver挂载，用freefilesync复制文件产生的，不知道哪个软件的原因
![Snipaste_2023-04-22_14-02-14](https://user-images.githubusercontent.com/2953848/233765831-6aac7959-e69d-4b4d-b5d5-36f32990ab97.png)

**Reply by hiforrest** [reply]
找到原因了，应该是 FreeFileSync 打开了「无风险的文件复制」导致的
